.github/workflows/release.yml

```diff
@@ -76,81 +63,3 @@ jobs:
           GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
           # GitHub sets this automatically
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      -
```

Comment:
by removing that, you also remove the publication on the public terraform, right? :/
Or did you set up a webhook (https://developer.hashicorp.com/terraform/registry/providers/publishing#publishing-to-the-registry)?

Reply:
Yes, we use the github webhook, so now each time a new release is created by `GoReleaser` on the repo it will be published automatically (: